plugins.txt

```diff
@@ -0,0 +1,18 @@
+ansible
+blueocean
+blueocean-i18n
+blueocean-config
+blueocean-events
+blueocean-web
+blueocean-commons
```

Comment:
I still think this is unnecessary bloat and doesn't really belong in this image. Nobody has asked for blueocean to be installed except for you.  So we would need to do some research to see if this is a thing that projects actually want and it would be in a separate PR anyway.